examples/jsm/loaders/EXRLoader.js

```diff
@@ -2656,6 +2687,19 @@ class EXRLoader extends DataTextureLoader {
 
 	}
 
+	/**
+	 * Controls whether the loader should expand output format to `RGBAFormat`. Defaults to `true`.
+	 *
+	 * @param {boolean} value - Whether to expand format or not.
+	 * @return {EXRLoader} A reference to this loader.
+	 */
+	setExpandedFormat( value ) {
+
+		this.shouldExpand = value;
+		return this;
+
+	}
```

Comment:
How about this, instead?
```js
setRGBAOutput( value ) {

    this.outputRGBA = value; // default true
    return this;

}
```

Or,
```js
setOutputType( value ) {

    this.outputType = value;  // default THREE.RGBAFormat; alt THREE.RedFormat
    return this;

}
```

Reply:
Whatever you guys prefer. But I would suggest against using `setOutputFormat`, as it can be confusing (only limited set of formats would be valid), and depending on future changes, the set of valid Formats could change based on input file.

Reply:
`setOutputFormat()` sounds good!